packages/amplify-graphql-transformer-core/src/types/types.ts

```diff
@@ -11,3 +11,13 @@ export type StackMeta = {
 };
 
 export type ApiStackType = 'models' | 'http' | 'predictions' | 'function' | 'openSearch' | 'rootStack';
+
+/**
+ *  Use it for all 'any's where we can't define the type, but doing a strict TypeScript conversion
+ */
+export type $TSAny = any; // eslint-disable-line  @typescript-eslint/no-explicit-any
+
+/**
+ * Use it for all object initializer usages: {}
+ */
+export type $TSObject = Record<string, $TSAny>;
```

Comment:
The other place where I would consider placing these is `amplify-graphql-transformer-interfaces` which we explicitly use to store common interfaces and types. Depending on what you think and others feel, I'll leave it upto you.

Reply:
I think it makes sense to move there.